src/utils/config.js

```diff
@@ -26,9 +26,12 @@ const DEFAULT_CONFIG = {
   languages: ['bash', 'json'],
   header_links: [],
   theme: 'default',
-  theme_custom: {
-    syntaxTheme: 'prism',
-    syntaxLineNumbers: false,
+  prefixTitles: true,
+  tableOfContents: true,
```

Comment:
Either use `table_of_contents`, or switch up the other config options to use camel case. Should stay consistent.